### PR TITLE
feat: Add e2e user invite test suite (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**

### CODE REVIEW

File: cypress.config.js
Comment (alexgrozav) [inline cypress.config.js:8]: For disabling retrying failed runs when developing.

File: cypress/e2e/14-sharing.cy.ts
Comment (MiloradFilipovic) [inline cypress/e2e/14-sharing.cy.ts:37]: Nit: Probably should update description

**Review by MiloradFilipovic — DISMISSED** [review]
Looks good.

File: cypress/e2e/14-sharing.cy.ts
Comment (alexgrozav) [reply cypress/e2e/14-sharing.cy.ts]: Good catch! Done.

**Review by MiloradFilipovic — DISMISSED** [review]
Re-approving after tests are fixed

**Review by MiloradFilipovic — APPROVED** [review]
Re-approve after merge

**Comment by janober** [comment]
Got released with `n8n@0.215.0`

**Pull request merged**
